app/bot/bot.go

```diff
@@ -17,6 +17,7 @@ import (
 //go:generate moq --out mocks/http_client.go --pkg mocks --skip-ensure . HTTPClient:HTTPClient
 //go:generate moq --out mock_interface.go . Interface
 //go:generate moq --out mocks/super_user.go --pkg mocks --skip-ensure . SuperUser:SuperUser
+//go:generate moq --out mocks/openai_client.go --pkg mocks --skip-ensure . OpenAIClient:OpenAIClient
```

Comment:
wrong file? The interface OpenAIClient not defined here, should be part of openai.go

Reply:
My logic: I wanted to have all `moq` calls at the same space. Using `moq` isn't clear for people who aren't familiar with go. There are all commands that have `--pkg mocks --skip-ensure` part.

Ideally, I'd like to have original `moq` command in the auto-generated files. This will make easy way to repeat it.

Reply:
Not sure I follow: `go generate` runs all the "go:generate" directives and regenerates all the mocks . No need to run anything manually and no need to be familiar with the moq too deep. Well, a developer has to have it installed, but this is about it.

Having an interface and moq command near this interface makes perfect sense to me